### PR TITLE
PPL-1240 : Added permission to access the site structure overview page.

### PR DESCRIPTION
**Pull request by zero2one**

### CODE REVIEW

File: og_sm_admin_menu/og_sm_admin_menu.module [hunk 1]
```diff
@@ -70,13 +73,27 @@ function og_sm_admin_menu_menu() {
...
 
   return $items;
 }
 
+/**
+ * Implements hook_og_permission().
+ */
+function og_sm_admin_menu_og_permission() {
+  return array(
+    OG_SM_ADMIN_MENU_PERMISSION_SITE_STRUCTURE => array(
```
Comment (MPParsley) [inline og_sm_admin_menu/og_sm_admin_menu.module:88]:
```suggestion
    OG_SM_ADMIN_MENU_PERMISSION_SITE_STRUCTURE => [
```

File: og_sm_admin_menu/og_sm_admin_menu.module [hunk 2]
```diff
@@ -70,13 +73,27 @@ function og_sm_admin_menu_menu() {
...
   );
 
   return $items;
 }
 
+/**
+ * Implements hook_og_permission().
+ */
+function og_sm_admin_menu_og_permission() {
+  return array(
```
Comment (MPParsley) [inline og_sm_admin_menu/og_sm_admin_menu.module:87]:
```suggestion
  return [
```

File: og_sm_admin_menu/og_sm_admin_menu.module [hunk 3]
```diff
@@ -70,13 +73,27 @@ function og_sm_admin_menu_menu() {
...
+    OG_SM_ADMIN_MENU_PERMISSION_SITE_STRUCTURE => array(
+      'title' => t('Administer Site structure'),
+      'description' => t('View the Site structure administration pages.'),
+      'roles' => array(OG_AUTHENTICATED_ROLE),
+      'default role' => array(OG_ADMINISTRATOR_ROLE),
+    ),
```
Comment (MPParsley) [inline og_sm_admin_menu/og_sm_admin_menu.module:93]:
```suggestion
    ],
```

File: og_sm_admin_menu/og_sm_admin_menu.module [hunk 4]
```diff
@@ -70,13 +73,27 @@ function og_sm_admin_menu_menu() {
...
+  );
```
Comment (MPParsley) [inline og_sm_admin_menu/og_sm_admin_menu.module:94]:
```suggestion
  ];
```

**Review by MPParsley — APPROVED** [review]

File: og_sm_admin_menu/og_sm_admin_menu.module (hunk 1 above)
Comment (zero2one) [reply og_sm_admin_menu/og_sm_admin_menu.module:88]: Drupal 7 ;-)

Comment (MPParsley) [reply og_sm_admin_menu/og_sm_admin_menu.module:88]: argh

**Pull request merged**